app/src/main/res/values/strings.xml

```diff
@@ -64,6 +64,7 @@
     <string name="delete_list">Delete List</string>
 
     <string name="list_name_input_speech_message">Tell us what you want to name your list</string>
+    <string name="name_input_speech_message_to_list">Tell us the name you want to add to %s\'s list</string>
```

Comment:
Can you change this to "Tell us what name you want to add" as mentioned in the issue: https://github.com/Gear61/Random-Name-Picker/issues/85

I think the list you're within is pretty clear since it's in the toolbar. It's not like when you clicked on a specific item within the list, of which there are many in the UI.

Also, there should be a edit_name_list_strings.xml file which is a better place for this resource.

Reply:
Sorry, after moving to the next notes you asked for and working with the other stuff like animation, I forgot the first note which is about the speech promote message text :)